tools/distpartitioning/parmetis_wrapper.py

```diff
@@ -24,6 +24,11 @@ def check_dependencies():
         mpi_install
     ), "Could not locate the following dependency: MPI. Please install it and try again."
 
+    dgl_path = os.environ.get("DGL_HOME")
+    assert os.path.isdir(
+        dgl_path
+    ), "Environment variable DGL_HOME not found. Please define the DGL installation path"
```

Comment:
`DGL repository path`?

Reply:
Yes. This is the DGL repository or DGL installation path. 

Typically, DGL_HOME is put in the '.profile' file, when I install DGL. 
What do you think about this change. This will allow us to run parmetis_wrapper.py from any location other than the dgl installation location.